src/lib/profile.js

```diff
@@ -49,6 +49,7 @@ function parseProfileData(profile, verifiedAccounts) {
 
   return {
     ...profile,
+    profileExists: Boolean(Object.keys(profile).length),
```

Comment:
Maybe we could early return in `getProfileForAccount` if profile doesn't exist and just return true here ?
Also lets order it.

Reply:
will do this in another pr when your fixes get merged